### DISCUSSION TITLE
Add a "New" button to the discussion view

### DISCUSSION THREAD

**Discussion by Zerotask** [community/community#5803] [Discussions]
If you click a specific issue, you will find a "New issue" button at the top right corner (on the issue title level)
![image](https://user-images.githubusercontent.com/20150243/134767723-6ca5b9ed-af66-42e5-a71f-fd57f002cacd.png)

I suggest adding the same behavior for the discussion view - add a "New discussion" button at the top right corner [marked as X].
If it's your own discussion, then it's next to the "edit" button.
![image](https://user-images.githubusercontent.com/20150243/134767769-45fbf505-7818-4e7b-b463-562685dd0a06.png)

This way it's easier to start discussing, since you don't have to do the extra step and click on the "Discussions" tab.

**Comment by polevaultweb** [comment]
Yes please, this would save me multiple clicks a day!
